app/src/main/java/io/transcend/samplesdk/MainActivity.java

```diff
@@ -25,19 +26,19 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         ImageView iconImageView = (ImageView) findViewById(R.id.iconImageView);
         iconImageView.startAnimation(getZoomOutAnimation());
-        String url = "https://transcend-cdn.com/cm/a3b53de6-5a46-427a-8fa4-077e4c015f93/airgap.js";
+        String url = "https://cdn.dev.trancsend.com/anotherminh/development/cm/66ded8fe-b10f-4f6c-9829-bf277bc28381/airgap.js";
```

Comment:
Can we use this bundle URL? And leave a comment saying this is the Managed Consent Testing bundle so we know which org it belongs to 😆 

https://transcend-cdn.com/cm/ 63b35d96-a6db-436f-a1cf-ea93ae4be24e/airgap.js;

Let's not use/commit anything other than prod bundles 🙏

Reply:
Updated!!